docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -0,0 +1,102 @@
+---
+id: plugin-types-usage
+title: Plugin types and usage
+description: Learn about the different types of Grafana plugins, their usage and configuration.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - usage
+  - provisioning
+  - provision
+  - configuration
+  - configure
+sidebar_position: 1
+---
+
+# Grafana plugin types and usage
+
+:::note
+
+This documentation discusses plugin configuration and usage for plugin developers. General information for plugin users is available at [Plugin management](https://grafana.com/docs/grafana/latest/administration/plugin-management/).
+
+::: 
+## Overview
+
+Grafana plugin development allows for many options depending on the type of user experience you want to create. You can find a plugin type whether you want to offer users a new way of visualizing data, a connection to a new database, or an integrated out-of-the-box experience. 
+
+The following table shows the plugin types currently supported by Grafana on the frontend and backend. 
+
+|                |                 |                |                            |
+| -------------- | :-------------: | :------------: | :------------------------: |
+| **Type**       | **Frontend** |   **Backend**  | **SDK-supported** |
+| datasource     |       Yes       | Yes (optional) |             Yes            |
+| app            |       Yes       | Yes (optional) |             Yes            |
+| panel          |       Yes       |       No       |             Yes            |
+
+## Data source plugins
+
+In plugin development, you can create new data source plugins to use Grafana with new databases, similar to existing plugins such as [MongoDB](https://grafana.com/grafana/plugins/grafana-mongodb-datasource/) or [Google BigQuery](https://grafana.com/grafana/plugins/grafana-bigquery-datasource/). Data source plugins may be added on the frontend and [the backend](./backend.md).
+
+### Usage of data-source plugins
+
+Create and configure a Grafana data source when you want to work with data provided by a third-party service for use in Grafana Dashboards, Explore, Alerting, and so on.
+
+Given any installed plugin of type `datasource`, you can create and configure any number of data sources per [Grafana organization](https://grafana.com/docs/grafana/latest/administration/organization-management/#about-organizations). After you create zero to infinity data sources, they are persisted in Grafana's database. 
+
+:::note
+
+To distinguish a Grafana data source from a data-source plugin, we sometimes refer to the latter as a _data-source instance_, that is, a configured Grafana data source with a `plugin id` type of `datasource`.
+
+:::
+
+
+### Provisioning of data sources
+
+Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/) to configure your data source plugins.
```

Comment:
Nit. I'm not sure just linking to the Grafana configure page would help plugin authors understand what this means. Did you specifically have https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#pluginplugin_id in mind?

Same applies to same section for apps

Reply:
Not exactly. Upon further reflection, I think it would be helpful to link both to the config file location and the specific plugin_id, like so:

Use the [Grafana configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#configuration-file-location) to configure your app `[`plugin_id`](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#pluginplugin_id).

What do you think?